services/thermal_mgr/thermal_mgr.c

```diff
@@ -43,18 +44,43 @@ void initThermalSystemManager(lm75bd_config_t *config) {
 
 error_code_t thermalMgrSendEvent(thermal_mgr_event_t *event) {
   /* Send an event to the thermal manager queue */
+  xQueueSend(thermalMgrQueueHandle, event, (TickType_t) 10);
```

Comment:
- [ ] check that the queue handle isn't null before use, returning invalid state
- [ ] check that the event isn't null
- [ ] use 0 for block time instead of 10 as we don't want to block in an isr